src/components/ui/AudioTrack/AudioTrack.module.css

```diff
@@ -55,18 +55,26 @@
 }
 
 .playToggle {
-  display: flex;
-  justify-content: center;
-  align-items: center;
   background: transparent;
   border: 0;
   color: inherit;
   cursor: pointer;
+
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: flex-start;
```

Comment:
@Yura33-dev Per [MDN Web Docs](https://www.w3.org/TR/css-flexbox-1/#justify-content-property), `flex-start` is the initial value of the CSS Flexbox `justify-content` property. We can safely remove this line as it won't affect our element. Unless I'm missing something 🤓 ?

Reply:
@naumch1k You are absolutely right! I will remove this property

Reply:
Also I have provided `aria-label` attribute for button.